web-admin/src/features/projects/github/GithubRepoSelectionDialog.svelte

```diff
@@ -0,0 +1,135 @@
+<script lang="ts">
+  import {
+    createAdminServiceGetProject,
+    createAdminServiceUpdateProject,
+    getAdminServiceGetGithubUserStatusQueryKey,
+  } from "@rilldata/web-admin/client";
+  import { GithubRepoUpdater } from "@rilldata/web-admin/features/projects/github/GithubRepoUpdater";
+  import {
+    AlertDialog,
+    AlertDialogContent,
+    AlertDialogDescription,
+    AlertDialogFooter,
+    AlertDialogHeader,
+    AlertDialogTitle,
+    AlertDialogTrigger,
+  } from "@rilldata/web-common/components/alert-dialog";
+  import { Button } from "@rilldata/web-common/components/button";
+  import Github from "@rilldata/web-common/components/icons/Github.svelte";
+  import Select from "@rilldata/web-common/components/forms/Select.svelte";
```

Comment:
A little out-of-scope of this PR, but it'd be nice to deprecate this in favor of [ShadCN's Select component](https://www.shadcn-svelte.com/docs/components/select)